subsystem/default_ingress_resource_test.go

```diff
@@ -338,6 +338,50 @@ var _ = Describe("default ingress", func() {
 		Expect(terraform.Apply()).To(BeZero())
 	})
 
+	It("Create cluster with default ingress - cluster_routes_hostname and cluster_routes_tls_secret_ref set to empty value", func() {
+		// Prepare the server:
+		server.AppendHandlers(
```

Comment:
If im not mistaken, You don't need all these server handlers. once you "apply" with an empty string, terraform should fail before hitting any API (due to the validation)